dashboard/test/ui/step_definitions/eyes_steps.rb

```diff
@@ -6,6 +6,10 @@
 
 When(/^I open my eyes to test "([^"]*)"$/) do |test_name|
   ensure_eyes_available
+
+  batch_name = test_name + " | " + ENV['BATCH_NAME']
+  @eyes.batch = Applitools::Base::BatchInfo.new(batch_name)
```

Comment:
I think I misunderstood the 'batch' capabilities at first. Will it actually group a whole run for us - where we can see individual tests by expanding on the page? ([source](http://support.applitools.com/customer/portal/articles/1665101-aggregating-grouping-several-tests-under-the-same-batch-))

![image](https://cloud.githubusercontent.com/assets/1615761/14970151/ae939668-107b-11e6-9b83-ed817339958e.png)

If so, that seems better.  To get that working, you might need to pass the second `started_at` parameter to `BatchInfo.new` ([see here](https://docs.omniref.com/ruby/gems/eyes_selenium/1.23.0/symbols/Applitools::BatchInfo)) as I suspect the start time is really the key for these.